com.unity.ml-agents.extensions/LICENSE.md

```diff
@@ -1,9 +1,204 @@
-com.unity.ml-agents.extensions copyright © 2020 Unity Technologies
+com.unity.ml-agents.extensions copyright © 2020 Unity Technologies ApS
```

Comment:
This is copied from com.unity.ml-agents/LICENSE.md (with the package name changed).